packages/ag-charts-react/project.json

```diff
@@ -36,6 +50,34 @@
         "format": ["cjs", "esm"]
       }
     },
+    "build:umd": {
+      "executor": "@nx/esbuild:esbuild",
+      "dependsOn": ["build:package"],
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "outputPath": "packages/ag-charts-react/dist/umd",
+        "outputFileName": "ag-charts-react",
+        "main": "packages/ag-charts-react/dist/index.cjs",
+        "tsConfig": "packages/ag-charts-react/tsconfig.lib.json",
+        "esbuildConfig": "esbuild.config.cjs",
+        "platform": "browser",
+        "target": "es6",
+        "bundle": true,
+        "thirdParty": true,
+        "skipTypeCheck": true,
+        "external": ["react", "react-dom", "react/jsx-runtime", "ag-charts-community"],
+        "format": ["cjs"],
+        "sourcemap": true
```

Comment:
How many of these do we need if we set the esbuildConfig?

Reply:
The esbuild config is setup for `ag-charts-community` and `ag-charts-enterprise` non-UMD modules, so these overrides are needed - for now I only want to encode the things we can't configure through Nx in those specialised Javascript config files.